### ISSUE TITLE
Windows

### ISSUE THREAD

**Issue by chetan1836** [ullasbharadwaj/secured_face_unlock#1]
Does this work with Windows 10 platform.

**Comment by ullasbharadwaj** [comment]
Hi, 
Thanks for creating this issue. 
Currently I have tested it only for Linux. When I find time, I would do it for Windows.

**Comment by chetan1836** [comment]
I tried to run on windows, but it didn't work. Anyways, thanks for the response

**Comment by ullasbharadwaj** [comment]
True. That's because I am using some Linux specific bindings. This can be solved. I will do it.
